espnet2/asvspoof/loss/oc_softmax_loss.py

```diff
@@ -27,30 +26,32 @@ def __init__(
 
     def forward(self, label: torch.Tensor, emb: torch.Tensor, **kwargs):
         """Forward.
+
         Args:
             label (torch.Tensor): ground truth label [Batch, 1]
             emb   (torch.Tensor): encoder embedding output [Batch, T, enc_dim]
         """
         emb = torch.mean(emb, dim=1)
-        w = torch.nn.functional.normalize(self.center, p=2, dim=1)
-        x = torch.nn.functional.normalize(emb, p=2, dim=1)
+        _ = torch.nn.functional.normalize(self.center, p=2, dim=1)
+        _ = torch.nn.functional.normalize(emb, p=2, dim=1)
```

Comment:
Are they OK?
I think we should keep `w` and `x`

Reply:
I did not want to change this file, but because the variables are assigned and never used, flake8 was raising errors. I will change back and add `#noqa`  at the end